templates/taxi/driver_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>Create driver</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+
+    <input type="submit" value="Submit" class="btn btn-primary">
+  </form>
```

Comment:
There should be a 'Cancel' button in addition to the 'Submit' button. This 'Cancel' button will lead to the previous page the user was on. You can implement it using a simple anchor tag with href set to the URL of the previous page.